### PR TITLE
Action updates

### PR DESCRIPTION
**Pull request by ANDREYDEN**
Changes all the actions to their batch versions

### CODE REVIEW

File: src/views/NewProduct.vue [hunk 1]
```diff
@@ -43,9 +43,9 @@ export default class NewProduct extends Vue {
 
   async removeExistingProduct(product: Product) {
```
Comment (alisondraV) [inline src/views/NewProduct.vue:44]: This function is not needed anymore, actually. `@remove`  at the top will never be emitted, so we can delete it.

File: src/views/FridgeSetup.vue [hunk 2]
```diff
@@ -57,7 +57,7 @@ export default class Fridge extends Vue {
 
   async addProductsToStorage() {
     for (const product of this.productsToAdd) {
-      await Firestore.instance.addProductToStorage(product);
+      await Firestore.instance.addToStorage([product]);
     }
```
Comment (alisondraV) [inline src/views/FridgeSetup.vue:61]: Shouldn't it just be `await Firestore.instance.addToStorage(this.productsToAdd)`?

File: src/utils/Firestore.ts [hunk 3]
```diff
@@ -142,6 +140,7 @@ export default class Firestore {
     const family = await CurrentFamily.instance.getCurrentFamily();
 
     const storageProductNames = family.storage.map(p => p.name);
+
```
Comment (alisondraV) [inline src/utils/Firestore.ts:143]: Why extra line?

File: src/utils/Firestore.ts [hunk 4]
```diff
@@ -40,71 +40,69 @@ export default class Firestore {
...
       .doc(family.id)
       .set(family);
   }
 
-  public async addToShoppingList(product: ProductDTO) {
-    if (product instanceof Product) {
-      product = product.toDTO();
-    }
-    if (await this.isProductInShoppingList(product)) return;
+  public async addToShoppingList(products: ProductDTO[]) {
```
Comment (alisondraV) [inline src/utils/Firestore.ts:97]: This is almost identical to the `addToStorage` function - we should consider combining them or extracting another function in the future.

**Review by alisondraV — CHANGES_REQUESTED** [review]
Left comments

File: src/utils/Firestore.ts (hunk 4 above)
Comment (ANDREYDEN) [reply src/utils/Firestore.ts:97]: oh, wait for it

Comment (ANDREYDEN) [reply src/utils/Firestore.ts:97]: it's gonna be great

**Comment by alisondraV** [comment]
In NewProduct on line 47 we can just pass this.location to the function. I would also suggest making ListName an enum.

**Review by alisondraV — APPROVED** [review]

**Pull request merged**